kernel/src/main/scala/cats/kernel/instances/uuid.scala

```diff
@@ -0,0 +1,19 @@
+package cats.kernel
+package instances
+
+import java.util.UUID
+
+package object uuid extends UUIDInstances
+
+trait UUIDInstances {
+  implicit val catsKernelStdOrderForUUID: Order[UUID] = new UUIDOrder
+}
+
+class UUIDOrder extends Order[UUID] {
```

Comment:
I wonder if you would be willing to add `fromComparable` to `object Order` similar to the `Show.fromToString`, and use it here. So, it would be a method like:
```scala
object Order {
  def fromComparable[T <: Comparable]: Order[T] = new Order[T] {
   ...
  }
}
```